src/components/patch/patch.tsx

```diff
@@ -0,0 +1,62 @@
+import { ExpansionPanel, ExpansionPanelDetails, ExpansionPanelSummary, Grid, Typography } from '@material-ui/core';
+import ExpandMoreIcon from '@material-ui/icons/ExpandMore';
+import * as React from 'react';
+import * as rest from "../../rest/interface";
+import '../../styles.css';
+import Variant from '../variant/variant';
+
+interface State {
+  APIClient: rest.Evergreen;
```

Comment:
This should be passed in by the parent, so it should be props

Reply:
Fixed